tests-e2e/go.mod

```diff
@@ -7,11 +7,11 @@ require (
 	github.com/argoproj/argo-cd/v2 v2.5.4
 	github.com/argoproj/gitops-engine v0.7.1-0.20221004132320-98ccd3d43fd9
 	github.com/openshift/api v3.9.1-0.20190916204813-cdbe64fb0c91+incompatible
-	github.com/redhat-appstudio/application-api v0.0.0-20230116140450-3ef8e0cd2c9e
+	github.com/redhat-appstudio/application-api v0.0.0-20230303141937-394e8c127f03
 	github.com/redhat-appstudio/managed-gitops/appstudio-controller v0.0.0
 	github.com/redhat-appstudio/managed-gitops/backend v0.0.0
 	github.com/redhat-appstudio/managed-gitops/backend-shared v0.0.0
-	github.com/redhat-appstudio/managed-gitops/cluster-agent v0.0.0
+	github.com/redhat-appstudio/managed-gitops/cluster-agent v0.0.0-00010101000000-000000000000
```

Comment:
I think this can just be `github.com/redhat-appstudio/managed-gitops/cluster-agent v0.0.0` (I don't actually know if there is a difference, but I'd prefer to stay on the side of caution. I tested that `go mod tidy` doesn't touch it.)